.circleci/config.yml

```diff
@@ -176,7 +176,8 @@ jobs:
           name: "Set module source to latest published release"
           working_directory: tests
           command: |
-            latest_release_tag="$(git tag --sort=-v:refname | head -1)"
+            # latest_release_tag="$(git tag --sort=-v:refname | head -1)"
+            export latest_release_tag="PLAT-6503-ish"
```

Comment:
This will be reverted once #84 is merged and a release is created based on it, prior to merging this.